pinot-spi/src/main/java/org/apache/pinot/spi/data/DateTimeFieldSpec.java

```diff
@@ -74,6 +75,10 @@ public DateTimeFieldSpec(String name, DataType dataType, String format, String g
       @Nullable Object sampleValue) {
     super(name, dataType, true);
 
+    // Override format to be "TIMESTAMP" for TIMESTAMP data type because the format is implicit
+    if (dataType == DataType.TIMESTAMP) {
```

Comment:
will this change people's schemas as they're stored in ZK, or just modify them every time they're loaded in ZK?

I almost wonder if we should go the route of not letting create/update happen with this rather than silently changing it so folks are aware

Reply:
This won't change existing schema. New uploaded/modified schema will be changed.
This reminds me that I do need to change the JSON setter so that create/update can be changed